modules/monitoring-configuring-a-persistent-volume-claim.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// monitoring/configuring-the-monitoring-stack.adoc
+
+[id='configuring-a-persistent-volume-claim-{context}']
+= Configuring a persistent volume claim
+
+For the Prometheus or Alertmanager to use a persistent volume (PV), you first need to configure a persistent volume claim (PVC).
+
+.Prerequisites
+
+* Make sure you have the necessary storage class configured.
+// FIXME add link, potentially https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/block_storage
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` ConfigMap:
++
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Put your PVC configuration for the component under `data/config.yaml`:
++
+[source,yaml,subs=quotes]
+----
+data:
+ config.yaml: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+   name: cluster-monitoring-config
+   namespace: openshift-monitoring
+  data:
+   config.yaml: |
+     *_component_*:
+       volumeClaimTemplate:
+         metadata:
+           name: *_PVC name_*
+         spec:
+           storageClassName: *_storage class_*
+           resources:
+             requests:
+               storage: *_40Gi_*
+----
++
+See the link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[Kubernetes documentation on PersistentVolumeClaims] for information on how to specify `volumeClaimTemplate`.
++
+For example, to configure a PVC that claims any configured {product-title} block PV as a persistent storage for Prometheus, use:
++
+[source,yaml,subs=quotes]
+----
+data:
```

Comment:
Should be
`apiVersion: v1
kind: ConfigMap
metadata:
  name: cluster-monitoring-config
  namespace: openshift-monitoring```